doc/source/conf.py

```diff
@@ -3,73 +3,110 @@
+        a_h_path = os.path.abspath(os.path.join('..', 'astropy_helpers'))
         if os.path.isdir(a_h_path):
             sys.path.insert(1, a_h_path)
 
-# -- Read the Docs Setup  -----------------------------------------------------
+    # If that doesn't work trying to import from astropy_helpers below will
+    # still blow up
+
+# Load all of the global Astropy configuration
+from astropy_helpers.sphinx.conf import *
+from sunpy.extern import six
+import sunpy
 
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
 
-if on_rtd:
-    os.environ['SUNPY_CONFIGDIR'] = '/home/docs/'
-    os.environ['HOME'] = '/home/docs/'
 
 # -- Download Sample Data -----------------------------------------------------
 
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
-# -- Shut up numpy warnings from WCSAxes
-
-import numpy as np
-np.seterr(invalid='ignore')
-
 # -- General configuration ----------------------------------------------------
 
+# If your documentation needs a minimal Sphinx version, state it here.
+needs_sphinx = '1.6'
 
-# Load all of the global Astropy configuration
-from astropy_helpers.sphinx.conf import *
+# To perform a Sphinx version check that needs to be more specific than
+# major.minor, call `check_sphinx_version("x.y.z")` here.
+check_sphinx_version("1.6")
 
-# If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '1.1'
+# The intersphinx_mapping in astropy_helpers.sphinx.conf refers to astropy for
+# the benefit of affiliated packages who want to refer to objects in the
+# astropy core.  However, we don't want to cyclically reference astropy in its
+# own build so we remove it here.
+#del intersphinx_mapping['astropy']
+
+# add any custom intersphinx for sunpy
+intersphinx_mapping.pop('h5py', None)
+intersphinx_mapping['astropy'] = ('http://docs.astropy.org/en/stable/', None)
+intersphinx_mapping['sqlalchemy'] = ('http://docs2.sqlalchemy.org/en/latest/', None)
+intersphinx_mapping['pandas'] = ('http://pandas.pydata.org/pandas-docs/stable/', None)
+intersphinx_mapping['skimage'] = ('http://scikit-image.org/docs/stable/', None)
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 exclude_patterns.append('_templates')
 
+# Add any paths that contain templates here, relative to this directory.
+if 'templates_path' not in locals():  # in case parent conf.py defines it
+    templates_path = []
+templates_path.append('_templates')
+
+
 # This is added to the end of RST files - a good place to put substitutions to
 # be used globally.
 rst_epilog += """
-"""
-
-# put auto generated api docs in the generated folder.
-automodapi_toctreedirnm = 'generated/api'
+.. SunPy
+.. _SunPy: http://sunpy.org
+.. _`SunPy mailing list`: https://mail.python.org/mailman/listinfo/sunpy
```

Comment:
is that correct? I'm not in that mailing list... I think it should be:
`SunPy mailing list`: http://groups.google.com/group/sunpy`